maint/local_python/binding_c.py

```diff
@@ -1012,7 +1012,10 @@ def process_handle(p):
                 pre_filters.append("if (%s_abi != NULL) {" % name)
                 pre_filters.append("    %s = &%s_i;" % (name, name))
                 pre_filters.append("}")
-                post_filters.append("if (ret == MPI_SUCCESS && %s_abi != NULL) {" % name)
+                cond = "%s_abi != NULL" % name 
+                if T != 'Comm':
```

Comment:
Why only `Comm`? This looks wrong and fishy. Either you do it for all MPI handle types, or for none of them.

Reply:
However, you may want to initialize `"MPI_%s %s_i = MPI_%s_NULL;" % (T, name, name.upper())` to make sure you never return garbage to the caller. Unless all internal routines already do set out handles to the NULL handle in case of an errors.

Reply:
That makes sense. I agree.